src/app/helps/help.jsx

```diff
@@ -39,7 +41,7 @@ export default function HelpCard({ help }) {
             flexDirection: "row",
           }}
         >
-          <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
+          {/* <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
```

Comment:
can you please remove the comments

Reply:
but why we are commenting this?

Reply:
@soufyanelf in UI you change the date by location !!
<img width="206" alt="image" src="https://github.com/BizTech-Morocco/mosa3ada.biztech.ma/assets/11611370/f841ccc4-af9a-47cd-9bd0-19141d440b91">